taxi_service/settings.py

```diff
@@ -108,6 +110,8 @@
 
 AUTH_USER_MODEL = "taxi.Driver"
 
+LOGIN_REDIRECT_URL = "/taxi/"
```

Comment:
https://stackoverflow.com/questions/58029272/django-redirect-user-after-login#:~:text=And%20in%20your%20settings%20file%3A
here is how it will work

Reply:
thanks, for more information. but this is all implemented on the basis of app- accounts, and we deleted this folder. I made a redirect "/": the error disappeared, the tests passed.